rightsDB.pm

```diff
@@ -33,20 +31,6 @@ sub new {
   return bless $self, $class;
 }
 
-sub ConnectToSdrDb
-{
-     my $db_user   = shift;
-     my $db_passwd = shift;
-     my $db_server = shift;
-     my $db_name   = shift;
-
-     my $sdr_dbh;
-     $sdr_dbh   = DBI->connect( "DBI:MariaDB:$db_name:$db_server", $db_user, $db_passwd,
-               { RaiseError => 0, AutoCommit => 1 } ) || die "Cannot connect: $DBI::errstr";
```

Comment:
Just noting that the move removes the `AutoCommit => 1`, but that should be fine as this code is only using the read-only connection (and we weren't previously using autocommit for the read-write stuff)